boards/arm/rcar_h3ulcb/rcar_h3ulcb_cr7_defconfig

```diff
@@ -9,12 +9,8 @@ CONFIG_RAM_CONSOLE=y
 CONFIG_FLASH_SIZE=0
 CONFIG_FLASH_BASE_ADDRESS=0
 CONFIG_GPIO=y
-CONFIG_GPIO_RCAR=y
 CONFIG_SERIAL=y
-CONFIG_UART_RCAR=y
 CONFIG_UART_CONSOLE=y
 CONFIG_UART_INTERRUPT_DRIVEN=y
 CONFIG_CAN=y
-CONFIG_CAN_RCAR=y
-CONFIG_I2C=y
-CONFIG_I2C_RCAR=y
+CONFIG_I2C=y
```

Comment:
Note: I2C should, in fact, be removed since it is not considered to be a minimal peripheral. Samples or applications can enable it as needed.